DSharpPlus.Rest/DiscordRestClient.cs

```diff
@@ -551,7 +551,7 @@ public async Task<DiscordMessage> EditMessageAsync(ulong channel_id, ulong messa
         {
             builder.Validate(true);
 
-            return await this.ApiClient.EditMessageAsync(channel_id, message_id, builder.Content, builder.Embed, builder.Mentions).ConfigureAwait(false);
+            return await this.ApiClient.EditMessageAsync(channel_id, message_id, builder.Content, builder.Embed, builder.Mentions, default).ConfigureAwait(false);
```

Comment:
I didn't mention this in the review, but you should pass `builder.Components` here.